src/app/view/components/dot-container-selector/dot-container-selector.component.html

```diff
@@ -10,7 +10,8 @@
             [totalRecords]="paginationService.totalRecords"
             [labelPropertyName]="['name', 'parentPermissionable.hostname']"
             placeholder="{{ dotMessageService.get('editpage.container.add.label') }}"
-            persistentPlaceholder="true">
+            persistentPlaceholder="true"
+            [multiple]="multiple">
```

Comment:
the value here is true

Reply:
it will depend, of what the user send.. in this case we are sending true in `dot-edit-layout-grid.component.html`